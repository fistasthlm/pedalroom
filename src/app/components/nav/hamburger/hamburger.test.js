import React from 'react';
import { shallow } from 'enzyme';
import Hamburger from './hamburger';

const setup = () => {
    const props = {
        toggleMenu: jest.fn(),
    };

    return shallow(<Hamburger {...props} />);
};

describe('Render', () => {
    it('should render component', () => {
        const wrapper = setup();

        expect(wrapper).toMatchSnapshot();
    });
});
