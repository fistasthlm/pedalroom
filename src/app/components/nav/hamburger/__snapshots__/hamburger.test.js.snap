// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render should render component 1`] = `
<div
  className="hamburger-button"
>
  <button
    className="toggle-button"
    id="navbar-toggle-btn-left"
    onClick={[MockFunction]}
  >
    <span
      className="icon-bar"
    />
    <span
      className="icon-bar"
    />
    <span
      className="icon-bar"
    />
  </button>
</div>
`;
